### ISSUE TITLE
Clear login/user data after logging out

### ISSUE THREAD

**Issue by Bushuo** [ElektraInitiative/PermaplanT#1137]
### Describe the Bug

Currently after logging out, the data necessary to authenticate with the identity provider is still cached in the application.
I think it should be necessary to login with username/password again after logging out.

### Steps to Reproduce the Problem

- log in.
- log out.
- click the login button again.

### Expected Result

- log in.
- log out.
- log in with username/password again.

### Actual Result

- immediately be logged in without having the possibility to change the user account. 

### PermaplanT version

master

### Operating System

doesnt matter

### Browser

doesnt matter

**Comment by markus2330** [comment]
Yes, I fully agree that it is a bug, you need to restart the browser (or open private window) to be able to login as someone else.

I don't think it is an urgent problem though, as devs can help themselves and only devs use several accounts. If someone disagrees (and is annoyed by it), please say so.
